front/docker/Dockerfile.nginx

```diff
@@ -23,18 +28,10 @@ COPY --from=test_data . /tests/data
 
 FROM nginx:alpine
 
-RUN apk add npm && \
-    npm install -g @import-meta-env/cli && \
-    rm /etc/nginx/conf.d/default.conf
```

Comment:
Note: previously we were deleting this file, and now we're keeping it. If it doesn't break anything, this is completely fine by me - just wondering if it matters at all. (Feel free to resolve.)

Reply:
Fixed